src/store/modules/product/ProductState.ts

```diff
@@ -1,6 +1,7 @@
 export default interface ProductState {
```

Comment:
I think we should make it ShipmentState and not ProductState.

Reply:
Done